.github/workflows/cicd-dev.yml

```
@@ -0,0 +1,78 @@
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.SUBMODULE_ACCESS_TOKEN }}
+          submodules: true
+
+      - name: Update submodule
+        run: |
+          git submodule update --remote --recursive
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)